src/Aspire.Hosting.Azure.Provisioning/Provisioners/AzureProvisioner.cs

```diff
@@ -109,11 +100,16 @@ private async Task ProvisionAzureResources(IConfiguration configuration, IHostEn
 
         Lazy<Task<(ResourceGroupResource, AzureLocation)>> resourceGroupAndLocationLazy = new(async () =>
         {
+            if (string.IsNullOrEmpty(_options.Location))
+            {
+                throw new MissingConfigurationException("An azure location/region is required. Set the Azure:Location configuration value.");
```

Comment:
Why do we throw this exception if you don't set `_options.Location`, but we never actually use `_options.Location`?